java/java.editor/src/org/netbeans/modules/editor/java/JavaCompletionCollector.java

```diff
@@ -1073,7 +1073,7 @@ private Completion createExecutableItem(CompilationInfo info, ExecutableElement
                     break;
                 }
                 if (!inImport && !memberRef && cnt == 0 && cs.spaceWithinMethodCallParens()) {
```

Comment:
I think I would merge the `insertTextParams` tests to existing `if`, where possible:
```suggestion
                if (!inImport && !memberRef && cnt == 0 && cs.spaceWithinMethodCallParens() && insertTextParams) {
```